src/Spout/Writer/Common/Entity/Row.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Box\Spout\Writer\Common\Entity;
+
+use Box\Spout\Writer\Common\Entity\Style\Style;
+use Box\Spout\Writer\Common\Manager\Style\StyleMerger;
+
+class Row
+{
+    /**
+     * The cells in this row
+     * @var array
+     */
+    protected $cells = [];
+
+    /**
+     * The row style
+     * @var null|Style
+     */
+    protected $style = null;
+
+    /**
+     * @var StyleMerger
+     */
+    protected $styleMerger;
+
+    /**
+     * Row constructor.
+     * @param array $cells
+     * @param Style|null $style
+     */
+    public function __construct(array $cells = [], Style $style = null)
```

Comment:
First I thought of this as a "Row Builder" not an entity. Hence the example without constructure params. But - I'd personally be fine with such a signature. I just like the flexibility here. But - when we add a RowBuilder  - this can change. I'd suggest ```(array $cells, Style $style = null)```